udata_front/views/gouvfr.py

```diff
@@ -132,12 +132,18 @@ def dataset_apis(ctx):
     return theme.render('dataset-apis.html', apis=dataset.extras.get(APIGOUVFR_EXTRAS_KEY))
 
 
-@template_hook('oauth_theme_content')
+@template_hook('oauth_authorize_theme_content')
 def oauth_theme_content(ctx):
     grant = ctx['grant']
     return theme.render('api/oauth_authorize.html', grant=grant)
 
 
+@template_hook('oauth_error_theme_content')
+def oauth_theme_content(ctx):
+    request = ctx['request']
+    return theme.render('api/oauth_error.html', error=request.args.get('error'))
```

Comment:
Could you test with an actual error?

Reply:
I test it by manually creating a URL with an error like `/en/oauth/error?error=Incorrect%20field` but I don't know how to make an actual oauth error going on this page. When I made an error with authorize parameters, the error is displayed in an empty page, not this template.